EliteDangerous/EliteDangerous/RoutePlotter.cs

```diff
@@ -29,18 +29,40 @@ public class RoutePlotter
         public Point3D Coordsto;
         public string FromSystem;
         public string ToSystem;
-        public int RouteMethod;
+        public Metric RouteMethod;
         public bool UseFsdBoost;
 
         // METRICs defined by systemclass GetSystemNearestTo function
-        public static string[] metric_options = {
-            "Nearest to Waypoint",
-            "Minimum Deviation from Path",
-            "Nearest to Waypoint with dev<=100ly",
-            "Nearest to Waypoint with dev<=250ly",
-            "Nearest to Waypoint with dev<=500ly",
-            "Nearest to Waypoint + Deviation / 2"
-        };
+        public static string GetMetricName(Metric metric)
```

Comment:
This needs replacing with a string[] and using the enum to look up the text.  Since the class has the enum, its safe to just use a string[] to find the text to go along with the enum